<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #252525;
    color: white;
}

QPushButton {
    background-color: #333;
    color: white;
    border: 2px solid #AA2020;
    border-radius: 5px;
    padding: 5px;
}

QPushButton:hover {
    background-color: #444;
}

QLineEdit, QComboBox, QDateEdit {
    background-color: #333;
    color: white;
    border: 2px solid #AA2020;
    border-radius: 3px;
    padding: 3px;
}

QTableWidget {
    background-color: #222;
    color: white;
    border: 2px solid #AA2020;
    gridline-color: #AA2020;
}

QHeaderView::section {
    background-color: #AA2020;
    color: white;
    padding: 5px;
    border: 1px solid #222;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tab_repairs">
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane {
	background-color: rgb(37, 37, 37);
    border: 2px solid #FF5C5C;
    background-color: #252525;
}

QTabBar::tab {
    background: #404040;
    color: white;
    border: 2px solid #FF5C5C;
    padding: 5px;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
}

QTabBar::tab:hover {
    background: #FF5C5C;
}

QTabBar::tab:selected {
    background: #D32F2F;
    font-weight: bold;
}
</string>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="RE">
       <property name="styleSheet">
        <string notr="true">#RE::pane {
	border-color: rgb(246, 211, 45);
    border: 2px solid #FF5C5C;
    background-color: #252525;
}

QTabBar::tab {
    background: #404040;
    color: white;
    border: 2px solid #FF5C5C;
    padding: 5px;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
}

QTabBar::tab:hover {
    background: #FF5C5C;
}

QTabBar::tab:selected {
    background: #D32F2F;
    font-weight: bold;
}
</string>
       </property>
       <attribute name="title">
        <string>Repairs - Expenses</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>search</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>by date</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>by type</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>matricule</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="view_more">
          <property name="text">
           <string>View Details</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QListWidget" name="RE_list_widget">
          <item>
           <property name="text">
            <string> id_service - date - cost </string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="idLineEdit">
          <property name="placeholderText">
           <string>Transaction ID</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="salaryLineEdit">
          <property name="placeholderText">
           <string>Salary</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="bonusLineEdit">
          <property name="placeholderText">
           <string>Bonus</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="propertyLineEdit">
          <property name="placeholderText">
           <string>Property ID</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="serviceLineEdit">
          <property name="placeholderText">
           <string>Service ID</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="add_transaction">
          <property name="text">
           <string>Add Transaction</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="edit_transaction">
          <property name="text">
           <string>Edit Transaction</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="delete_transaction">
          <property name="text">
           <string>Delete Transaction</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="3">
         <widget class="QListWidget" name="transactionListWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="SM">
       <attribute name="title">
        <string>Salary Management</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="3">
         <widget class="QComboBox" name="comboBox_3">
          <item>
           <property name="text">
            <string>by mat</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>by full_name</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>by post</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>search</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QListWidget" name="listWidget">
          <item>
           <property name="text">
            <string>employee_mat  -  full_name - post - basic_salary - bonus_or_panelty - +hours  - date</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="delete_sm_btn">
          <property name="text">
           <string>delete</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4" colspan="2">
         <widget class="QListWidget" name="listWidget_4">
          <item>
           <property name="text">
            <string>fich de pay </string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>matricule :</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>full_name :</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>post :</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>date :</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>basic salary :</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>experience :</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>+hours :</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>penalty :</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>total salaire :</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QPushButton" name="pushButton_8">
          <property name="text">
           <string>print</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>display formul</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="stat">
       <attribute name="title">
        <string>stat</string>
       </attribute>
       <widget class="QChartView" name="chartView" native="true">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>500</width>
          <height>300</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="printstat">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>380</y>
          <width>119</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Print</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="arduino">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <widget class="QLineEdit" name="welcome">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>110</y>
          <width>311</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>printstat</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>printStatistics()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>printStatistics()</slot>
 </slots>
</ui>
